### ISSUE TITLE
Please add puppet 4 compatibility

### ISSUE THREAD

**Issue by kshcherban** [puppet-community/puppet-mcollective#278]
Hello,

Seems that current module is not compatible with puppet >= 4.
For example:
* package puppet-agent replaces mcollective, mcollective-common packages
* config directories have moved to /etc/puppetlabs/mcollective
* plugin directories seem to be moved into /opt/puppetlabs/mcollective

**Comment by jorhett** [comment]
This project is: https://github.com/jorhett/puppet-mcollective

**Comment by jyaworski** [comment]
Yes, I concur with @jorhett that implementing #279 should solve this.
